compiler/noirc_frontend/src/hir/mod.rs

```diff
@@ -50,6 +50,8 @@ pub struct Context<'file_manager, 'parsed_files> {
     pub parsed_files: Cow<'parsed_files, ParsedFiles>,
 
     pub package_build_path: PathBuf,
+
+    pub formal_verification: bool,
```

Comment:
Let's rename this to `enable_formal_verification` or `perform_formal_verification`. This way it would be clearer that this is a bool. At the moment, without context, it could be some sort of an artifact.

Reply:
Done.